src/main/java/chat/commands/roleDistribution/DeleteRole.java

```diff
@@ -0,0 +1,90 @@
+package chat.commands.roleDistribution;
+
+import chat.commands.Command;
+import sx.blah.discord.api.IDiscordClient;
+import sx.blah.discord.handle.obj.*;
+import sx.blah.discord.util.MessageBuilder;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.List;
+
+/**
+ * Created by Dries on 5/07/2017.
```

Comment:
The docstring for the class should include a explanation for the purpose of the class.